pkg/actuators/machine/actuator.go

```diff
@@ -376,20 +384,35 @@ func (a *Actuator) Update(context context.Context, cluster *clusterv1.Cluster, m
 		return fmt.Errorf(errMsg)
 	}
 
+	if len(runningInstances) == 0 {
```

Comment:
can't we `update()` method be reduced to replace `getRunningInstances` with `getExistingInstances`?